docs/pages/access-controls/access-request-plugins/ssh-approval-jira.mdx

```diff
@@ -1,224 +1,770 @@
 ---
-title: Access Requests using Jira and Teleport
-description: How to configure Access Request approval using Jira and Teleport
+title: Teleport Access Requests with Jira
+description: How to set up the Teleport Jira plugin to notify users when another user requests elevated privileges.
 ---
 
-This guide will talk through how to set up Teleport with Jira. Teleport's Jira
-integration allows you to treat Teleport access and permission requests using
-Jira tickets.
+This guide explains how to set up the Teleport Access Request plugin for Jira.
+Teleport's Jira integration allows you to manage Teleport Access Requests as
+Jira issues.
+
+The Teleport Jira plugin synchronizes a Jira project board with the Access
+Requests processed by your Teleport cluster. When you change the status of an
+Access Request within Teleport, the plugin updates the board. And when you
+update the status of an Access Request on the board, the plugin notifies a Jira
```

Comment:
FYI - the webhook server is disabled in cloud-hosted plugins, due to the added complexity of trying to get the callback requests back through the Teleport proxy to an arbitrary server inside the cluster. 

It's more of an issue for my #30254 PR, but it would be a big help if things are structured so that any mentions of the webhook can be easily omitted in the `cloud` scope

Reply:
The page now includes a partial that refers Cloud users to the Web UI, which hopefully removes some of the confusion.